src/Raven.Server/Documents/Sharding/Commands/ShardedQueryCommand.cs

```diff
@@ -10,7 +11,7 @@ public class ShardedQueryCommand : ShardedBaseCommand<QueryResult>
 {
     private readonly string _indexName;
 
-    public ShardedQueryCommand(ShardedDatabaseRequestHandler handler, BlittableJsonReaderObject content, string indexName) : base(handler, Commands.Headers.Sharded, content)
+    public ShardedQueryCommand(ShardedDatabaseRequestHandler handler, HttpMethod method, BlittableJsonReaderObject content, string indexName) : base(handler, method, Commands.Headers.Sharded, content)
```

Comment:
can we kill this command?

Reply:
I'm doing this in https://github.com/ravendb/ravendb/pull/14725